comid/rawvalue.go

```diff
@@ -41,6 +43,22 @@ func (o RawValue) GetBytes() ([]byte, error) {
 	}
 }
 
+func (o *RawValue) Equal(r *RawValue) bool {
```

Comment:
This may be a bit trickier than expected.

`Rawvalue` comparison also needs an optional mask to filter the "unimportant" bits (i.e., those that don't matter for comparison).  See [CoRIM](https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#section-5.1.4.1.4.2-5.5.1) for more details.

Besides, you'd compare it against a simple `[]byte` rather than another `RawValue`.

Therefore, the interface should look like:

```golang
func (o RawValue) Equal(other []byte, mask *[]byte) bool
```

This doesn't take into consideration the fact that a better way to model the `RawValue` type would be to include the optional mask alongside the value, which is not what we do ATM.  If we did that, then:

```golang
func (o RawValue) Equal(other []byte) bool
```